app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topics.controller');
+
+const app = express();
+
+app.use(express.json())
+
+app.get('/api', (req, res) => {
```

Comment:
this health check of your server should exist at a different endpoint as you will need this endpoint to do something else in task 3!